CONTRIBUTING.md

```diff
@@ -1,5 +1,23 @@
 # Contributing Guidelines
 
+### Commit Messages
+
+If a pull-request addresses a Jira ticket, for a single-commit PR, prefix
+the subject line with the ticket ID.  (For a multi-commit PR, we will add
+the ID later when we squash or merge it.)
+
+> CXX-883 Add commit message conventions to CONTRIBUTING.md
+
+Capitalize subject lines and don't use a trailing period.  Keep the subject
+under 60 characters long.  Use active voice!  Imagine this preamble to get
```

Comment:
Weak vote for "at most 70 characters" instead of "under 60 characters", since 71 characters is the length that GitHub will start to truncate the commit message when displaying the commit (e.g. see https://github.com/mongodb/mongo-cxx-driver/commits/master).